src/rviz/default_plugin/camera_display.cpp

```diff
@@ -34,6 +34,7 @@
 #include <OgreRectangle2D.h>
```

Comment:
We should remove the changes that are from https://github.com/ros-visualization/rviz/pull/1532 and discuss them there instead.